code/modules/mob/living/simple_mob/combat.dm

```diff
@@ -2,7 +2,7 @@
 /mob/living/simple_mob/proc/attack_target(atom/A)
 	set waitfor = FALSE // For attack animations. Don't want the AI processor to get held up.
 
-	if(!A.Adjacent(src))
+	if(!A.Adjacent(src) || isbelly(src.loc)) //VOREStation Edit
```

Comment:
TBH You might want to just return at the start of their handle_stance stuff, it will save CPU, unless there is something else you want them to be doing in bellies